src/components/Dashboard/AdminStatistics/AdminDetailedView/index.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/forbid-prop-types */
+import PropsTypes from 'prop-types';
+import { ComponentLayout } from 'layouts';
+import { latestReportColumns } from 'shared/objects/Users';
+import { Table } from 'components/AntDesign';
+
+export default function AdminDetailedView({ rows }) {
+  return (
+    <ComponentLayout heading="Latest Report ">
+      <Table
+        columns={latestReportColumns}
+        dataSource={rows.map(((row) => ({
+          title: row.title,
+          watchedTimeFrom: row.watchedTimeFrom,
+          watchedTimeTo: row.watchedTimeTo,
+          owedRevenue: `${row.owedRevenue.slice(0, 5)} £`,
```

Comment:
FYI, the price tag should be ahead of its number not after, we say `$12` or `12 USD` not `12$` 
```suggestion
          owedRevenue: ` £${row.owedRevenue.slice(0, 5)}`,
```